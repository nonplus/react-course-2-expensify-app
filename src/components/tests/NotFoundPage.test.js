import { shallow } from "enzyme";
import React from "react";
import NotFoundPage from "../NotFoundPage";

describe("NotFoundPage", () => {
  it("should render page correctly", () => {
    const wrapper = shallow(<NotFoundPage />);
    expect(wrapper).toMatchSnapshot();
  });
});
