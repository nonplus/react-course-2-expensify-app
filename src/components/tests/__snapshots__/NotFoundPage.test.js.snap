// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NotFoundPage /> should render page correctly 1`] = `
<div>
  404 - 
  <Link
    replace={false}
    to="/"
  >
    Go home
  </Link>
</div>
`;
